src/styles/_catalog.scss

```
@@ -14,9 +14,18 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition: transform 300ms;
+    transition-duration: 300ms;
```

If you set duration to transition property, you can delete transition-duration
```suggestion
    transition: transform 300ms;
```

---

Also add color to transition

---

transition: transfrom 300ms color 300ms;